storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -33,6 +33,12 @@ include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 include::modules/persistent-storage-csi-vsphere-stor-policy.adoc[leveloffset=+1]
 
+[NOTE]
+====
+If you want to update your cluster to {product-title} v4.10 or later and have previously installed an upstream vSphere CSI Operator Driver, you must remove the upstream driver before updating the cluster.
```

Comment:
```suggestion
If you want to update your cluster to {product-title} 4.10 or later and have previously installed an upstream vSphere CSI Operator Driver, you must remove the upstream driver before updating the cluster.
```
Re: [Repo guidlines](https://github.com/openshift/openshift-docs/blob/5f567ca2671caf897bacdad28dfe36e6a1ee288b/contributing_to_docs/doc_guidelines.adoc#product-name-and-version)